kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/TopicEncryptionST.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.strimzi.api.kafka.model.Kafka;
+
+import io.kroxylicious.systemtests.extensions.KroxyliciousExtension;
+import io.kroxylicious.systemtests.installation.kroxylicious.Kroxylicious;
+import io.kroxylicious.systemtests.installation.vault.Vault;
+import io.kroxylicious.systemtests.resources.vault.KubeVaultTestKmsFacade;
+import io.kroxylicious.systemtests.resources.vault.KubeVaultTestKmsFacadeFactory;
+import io.kroxylicious.systemtests.steps.KafkaSteps;
+import io.kroxylicious.systemtests.steps.KroxyliciousSteps;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaNodePoolTemplates;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaTemplates;
+import io.kroxylicious.systemtests.utils.NamespaceUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@ExtendWith(KroxyliciousExtension.class)
+class TopicEncryptionST extends AbstractST {
```

Comment:
Rightly or wrongly, we've been using the name EnvelopeEncryption in the code and the tests.  The system tests should be consistent with the rest of the code.  

```suggestion
class EnvelopeEncryptionST extends AbstractST {
```